### PR TITLE
Complete scripts for the population tables

### PR DESCRIPTION
**Pull request by jackieHubc**

### CODE REVIEW

**Comment by jackieHubc** [comment]
Orders number 1,2,3,4,5,7,9 are delivered, orders number 6,8,10 are cancelled. Orders number 10,11,12 are neither delivered nor cancelled.

**Comment by omzmarlon** [comment]
Nice. Then just make some of them not taken by any senders

**Comment by jackieHubc** [comment]
Adding three orders whose sender name fields are null. Right now the table should have 16 rows in total.

File: src/main/resources/schemas/populate_orders.sql [hunk 1]
```diff
@@ -35,4 +35,13 @@ INSERT INTO order_info(sender_name,receiver_name,restaurant_name,order_cost,deli
 VALUES ('DanielWU', 'jasper_c', 'pizza_g', 35.00, 4.00, '2017-11-14 19:05:00', '2017-11-14 19:25:00','Ponderosa Oak, 2075 Lower Mall');
 
 INSERT INTO order_info(sender_name,receiver_name,restaurant_name,order_cost,deliver_tip,order_time,deadline,delivery_location)
-VALUES ('jasper_c', 'DanielWU', 'pizza_g', 28.00, 3.00, '2017-11-11 11:11:11', '2017-11-11 11:21:00','Hugh Dempster Pavilion, 6245 Agronomy Rd');
\ No newline at end of file
+VALUES ('jasper_c', 'DanielWU', 'pizza_g', 28.00, 3.00, '2017-11-11 11:11:11', '2017-11-11 11:21:00','Hugh Dempster Pavilion, 6245 Agronomy Rd');
```
Comment (omzmarlon) [inline src/main/resources/schemas/populate_orders.sql]: sender_name is still not null?

Comment (omzmarlon) [reply src/main/resources/schemas/populate_orders.sql]: aaaa my bad. it is null

**Pull request merged**